rdrf/rdrf/urls.py

```diff
@@ -291,7 +284,9 @@ def proms_only(url_pattern):
     re_path(r'^questionnaireconfig/(?P<form_pk>\d+)/?$',
             form_view.QuestionnaireConfigurationView.as_view(), name='questionnaire_config'),
 
-    re_path(r'api/familylookup/(?P<reg_code>\w+)/?$', FamilyLookup.as_view(), name="family_lookup"),
+    # Disabled as no registries use Family Linkage currently. Make sure it is secure if it needs to be re-enabled!
+    # re_path(r'api/familylookup/(?P<reg_code>\w+)/?$', FamilyLookup.as_view(), name="family_lookup"),
+
     re_path(r'api/patientlookup/(?P<reg_code>\w+)/?$', PatientLookup.as_view(), name="patient_lookup"),
```

Comment:
This one allows any patient to look up all the other patients by name  in the same WG.
Used only in questionnaires, so please disable with a similar comment as above.
ie.not used, review security if needed again.